.gitlab/package_build/deb.yml

```diff
@@ -64,6 +64,10 @@
     - $S3_CP_CMD $OMNIBUS_PACKAGE_DIR/datadog${FLAVOR:+-$FLAVOR}-agent_*_${PACKAGE_ARCH}.deb $S3_ARTIFACTS_URI/$DESTINATION_DEB
     - $S3_CP_CMD $OMNIBUS_PACKAGE_DIR/datadog${FLAVOR:+-$FLAVOR}-agent-dbg_*_${PACKAGE_ARCH}.deb $S3_ARTIFACTS_URI/$DESTINATION_DBG_DEB
     - !reference [.upload_sbom_artifacts]
+  variables:
+    KUBERNETES_MEMORY_REQUEST: "6Gi"
+    KUBERNETES_MEMORY_LIMIT: "12Gi"
```

Comment:
Is there any particular reason why we're choosing these memory values? Have you checked how much memory jobs on your branch take on average?

Reply:
any relation with the [study](https://docs.google.com/spreadsheets/d/1XmicdhhJCn6fMCPqFlLDfccR5rAJM8ed5-KE7gPNCK4/edit?usp=sharing) done few time ago?

Reply:
I don't think it applies here, build jobs have different requirements (and memory usage patterns) than unit test jobs.